addons/l10n_fr/data/tax_report_data.xml

```diff
@@ -962,7 +962,7 @@
                             <record id="tax_report_16" model="account.report.line">
                                 <field name="name">16 - Total gross VAT due</field>
                                 <field name="code">box_16</field>
-                                <field name="aggregation_formula">box_08_taxe.balance + box_09_taxe.balance + box_9B_taxe.balance + box_10_taxe.balance + box_11_taxe.balance + box_13_taxe.balance + box_14_taxe.balance + box_T1_taxe.balance + box_T2_taxe.balance + box_T3_taxe.balance + box_T4_taxe.balance + box_T5_taxe.balance + box_T6_taxe.balance + box_T7_taxe.balance + box_P1_taxe.balance + box_P2_taxe.balance + box_I1_taxe.balance + box_I2_taxe.balance + box_I3_taxe.balance + box_I4_taxe.balance + box_I5_taxe.balance + box_I6_taxe.balance + box_15.balance + box_5B.balance</field>
+                                <field name="aggregation_formula">box_08_taxe.balance_from_tags + box_09_taxe.balance + box_9B_taxe.balance + box_10_taxe.balance + box_11_taxe.balance + box_13_taxe.balance + box_14_taxe.balance + box_T1_taxe.balance + box_T2_taxe.balance + box_T3_taxe.balance + box_T4_taxe.balance + box_T5_taxe.balance + box_T6_taxe.balance + box_T7_taxe.balance + box_P1_taxe.balance + box_P2_taxe.balance + box_I1_taxe.balance + box_I2_taxe.balance + box_I3_taxe.balance + box_I4_taxe.balance + box_I5_taxe.balance + box_I6_taxe.balance + box_15.balance + box_5B.balance</field>
```

Comment:
isn't it an issue for the other taxes ?

In the ticket, they talk about a 20% taxes (invoice and bill)

But what about an invoice 20% and a bill 5.5%

Reply:
For the tax lines related to the bills (box_23), it seems that there is no issue. However, for some reason, invoices with 20% goods (services) do not appear at all in the tax report, but I am not sure if this is intended.